guide/17-geocoding-guides/part6_custom_geocoders.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>When the <code>geocoder</code> parameter is not applied, the <code>geocode()</code> function adopts the first registered geocoding service with the currently active <code>GIS</code>; when specified, <code>geocode()</code> uses the custom geocoder input with the parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>